cupy/statistics/order.py

```diff
@@ -113,4 +114,114 @@ def nanmax(a, axis=None, out=None, keepdims=False):
 # TODO(okuta): Implement ptp
 
 
-# TODO(okuta): Implement percentile
+def percentile(a, q, axis=None, out=None, interpolation='linear',
+               keepdims=False):
+    """Compute the qth percentile of the data along the specified axis.
```

Comment:
Compute -> Computes
qth -> q-th (?)

Reply:
qth was actually taken from the NumPy docs, but this looks better. Thank you, fixed.